### PR TITLE
Migration of picking_line_description

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rename it to _stock_move_description_ or _stock_picking_line_description_ :wink:

File: picking_line_description/stock_move.py [hunk 1]
```diff
@@ -59,10 +58,6 @@ def onchange_product_id(
                 product_obj = self.pool.get('product.product')
                 product = product_obj.browse(
                     cr, uid, prod_id, context=context)
-                if (
-                        product and
-                        product.description and
-                        'value' in res
-                ):
+                if (product.description and 'value' in res):
```
Comment (pedrobaeza) [inline picking_line_description/stock_move.py]: You should check the existence on value indepently from the other thing:

```
if product.description:
    if 'value' not in res:
       res['value'] = {}
    res['value']['name'] = product.description
```

**Comment by tafaRU** [comment]
@pedrobaeza I renamed the module, now you can review it. Thanks!

**Comment by tafaRU** [comment]
@pedrobaeza correction: I noticed some strings still refers to `pickin lines` instead of `stok moves`.  I'm fixing them.

**Comment by tafaRU** [comment]
@pedrobaeza done.

**Comment by pedrobaeza** [comment]
Travis error unrelated

:+1:

File: stock_move_description/__openerp__.py [hunk 2]
```diff
@@ -20,22 +19,15 @@
...
-Settings --> Configuration --> Warehouse --> Products
-    """,
-    'author': "Agile Business Group,Odoo Community Association (OCA)",
+    'author': "Agile Business Group, Odoo Community Association (OCA)",
     'website': 'http://www.agilebg.com',
     'license': 'AGPL-3',
     'depends': [
-        'stock',
+        'stock_account',
+        'delivery',
```
Comment (eLBati) [inline stock_move_description/__openerp__.py]: Why delivery?

Comment (tafaRU) [inline stock_move_description/__openerp__.py]: Fixed, thanks!

File: stock_move_description/__openerp__.py [hunk 3]
```diff
@@ -20,22 +19,14 @@
...
-
-Settings --> Configuration --> Warehouse --> Products
-    """,
-    'author': "Agile Business Group,Odoo Community Association (OCA)",
+    'author': "Agile Business Group, Odoo Community Association (OCA)",
     'website': 'http://www.agilebg.com',
     'license': 'AGPL-3',
     'depends': [
-        'stock',
+        'stock_account',
```
Comment (eLBati) [inline stock_move_description/__openerp__.py]: Why stock_account instead of just stock? :sweat_smile:

Comment (tafaRU) [inline stock_move_description/__openerp__.py]: @eLBati fixed.

**Comment by eLBati** [comment]
thanks :+1:

**Comment by yvaucher** [comment]
Restarted travis as tests were fixed in #111

**Comment by yvaucher** [comment]
@tafaRU can you rebase to trigger the right travis build (or at least `git commit --amend` you last commit), travis might fail as restarting it started again the old jobs

**Comment by tafaRU** [comment]
@yvaucher just rebased on top.

**Comment by yvaucher** [comment]
Much better we now have the 5 jobs with splitted test instead of 3

**Comment by tafaRU** [comment]
> Much better we now have the 5 jobs with splitted test instead of 3

yes, definetely!

@yvaucher : PR looks finally green and you could merge it. Thanks in advance.

**Pull request merged**